benchmarking/nursery/benchmark_automl/benchmark_main.py

```diff
@@ -133,10 +140,10 @@ def parse_args(methods: dict, benchmark_definitions: dict):
         help="number of brackets",
     )
     parser.add_argument(
-        "--start_seed",
+        "--num_samples",
         type=int,
-        default=0,
-        help="first seed to run (if run_all_seed)",
+        default=50,
+        help="Number of samples for Hyper-Tune distribution",
     )
```

Comment:
Could you remove this one given that this file is agnostic to the method?
If removing is too complicated, we can have a duplicate which just imports `get_transfer_learning_evaluations`.

Reply:
Good point